Building Wallet example (from workspace) - throws <unknown>:0: error: could not build Objective-C module 'crypto' - 'openssl/e_os2.h' file not found

![screen shot 2018-02-05 at 11 40 47 am](https://user-images.githubusercontent.com/289994/35816726-7b724754-0a69-11e8-826b-76cb861716ee.png)

I'm targeting ios 11 - (not ios 11.2)
seems like there's a specific openssl version required.
https://github.com/ZBK1nger/bither-iOS/blob/fd9646362483216b580f6f41151845ef4abb7608/Podfile

/Users/johndpope/Documents/cryptoWorkspace/BitcoinKit/Libraries/crypto/include/openssl/bn.h:129:11: error: 'openssl/e_os2.h' file not found
# include <openssl/e_os2.h>

Errors Installing the "cryptography" Package
If you see the error shown below on OS X, then there was an issue installing the cryptography package.

build/temp.macosx-10.10-x86_64-2.7/_openssl.c:423:10: fatal error: 'openssl/e_os2.h' file not found
    #include <openssl/e_os2.h>
To fix this issue, try upgrading Xcode Command Line Tools:

$ xcode-select --install
Then, make sure that openssl is installed and re-link it:

$ brew install openssl
$ brew link openssl --force


https://github.com/blockstack-packages/blockstack-profiles-py/blob/8d76651959847aa49c8e1d0714da4407c19c0cd4/troubleshooting.md

found this - it's out of date.

➜  BitcoinKit git:(btc) ✗ brew link --force openssl
Warning: Refusing to link: openssl
Linking keg-only openssl means you may end up linking against the insecure,
deprecated system OpenSSL while using the headers from Homebrew's openssl.
Instead, pass the full include/library paths to your compiler e.g.:
  -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib

--- a/Examples/Wallet/Wallet.xcodeproj/project.pbxproj
+++ b/Examples/Wallet/Wallet.xcodeproj/project.pbxproj
@@ -338,6 +338,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Carthage/Build/iOS",
 				);
+				HEADER_SEARCH_PATHS = "$(PROJECT_DIR)/../../Libraries/crypto/include";
 				INFOPLIST_FILE = Wallet/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.kishikawakatsumi.Wallet;
@@ -357,6 +358,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Carthage/Build/iOS",
 				);
+				HEADER_SEARCH_PATHS = "$(PROJECT_DIR)/../../Libraries/crypto/include";
 				INFOPLIST_FILE = Wallet/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.kishikawakatsumi.Wallet;

